bin/all_sky_search/pycbc_add_statmap

```diff
@@ -72,6 +74,15 @@ files = [h5py.File(n, 'r') for n in args.statmap_files]
 
 f = h5py.File(args.output_file, "w")
 
+# For a few things, we will want singles and coinc files separated
```

Comment:
I'm not actually seeing what the comment is referring to. I think this piece of code doesn't separate singles vs coinc files .. so the connection with the comment would have to be clarified